### DISCUSSION TITLE
How can I disable the styles?

### DISCUSSION THREAD

**Discussion by alissa-tung** [vincjo/datatables#119] [Q&A]
Hello, thank you for developing this library!

I noticed that in the website we declared that this is a headless solution, and my question is that from the `Datatable.svelte`, there are some [styles](https://github.com/vincjo/datatables/blob/d3d2238d3ae1dcc8526924b4a7b26c42dd29fb9c/src/lib/src/shared/Datatable.svelte#L43) inside the component. I am thinking about how can I disable the styles bring by the library.

**Comment by vincjo** [comment]
Hello,

I'm a bit stuck with this issue: how to propose the golden mean between ready-to-use components or 100% headless which requires more work to start with.

Currently the guideline was to implement the api (DataHandler (v1) / TableHandler (v2)) with your own components.

I agree that the `<Datatable>` would be more usable with a "headless" option to remove css.

I'm open for any suggestions, I will work on it before the release of V2
